css/index.css

```diff
@@ -143,4 +142,114 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/* HEADER NAVIGATION AND TOP CONTENTCODE */
+header nav {
+    margin-top: 40px;
+    margin-bottom: 40px;
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+}
+
+nav a {
+    text-decoration: none;
+    color: black;
+    margin-right: 50px;
+}
+
+.top-content {
+    margin-top: 20px;
+}
+
+/* BOX COLORING  */
+.one {
+    background-color: teal;
+}
+
+.two {
+    background-color: gold;
+}
+
+.three {
+    background-color: cadetblue;
+}
+
+.four {
+    background-color: coral;
+}
+
+.five {
+    background-color: crimson;
+}
+
+.six {
+    background-color: forestgreen;
+}
+
+.seven {
+    background-color: darkorchid
+}
+
+.eight {
+    background-color: hotpink;
+}
+
+.nine {
+    background-color: indigo;
+}
+
+.ten {
+    background-color: dodgerblue;
+}
+
+/* ABOUT SECTIONNNNNNNNNNNNNNNNNN BEGINNNNNNNNNNNNNNNS*/
+
+.theme {
+    margin: 30px;
+}
+
+/* ALL THINGS MIDDLE SECTIONNNNNNNNNNNNNNNNNN */
+
+.heart {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    border-top: 1px dashed black;
+    margin-top: 20px;
+}
+
+.heart section {
+    width: 380px;
+    border-bottom: 1px dashed black;
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+.heart h2 {
+    margin-top: 10px;
+}
+
+button {
+	background: white;
+	border-radius: 5px;
+	border: 2px solid black;
+	height: 40px;
+	width: 125px;
+    margin-bottom: 20px;
+    margin-top: 20px;
+}
+
+button:hover {
+    background: black;
+    color: white;
+    transform: rotate(-720deg);
+    height: 15%;
+    width: 15%;
+    transition: 3s ease-out;
+
+}
+
+/* LETS WORK TOGETHERRRRRRRRRRRRRRRRRRRRRRRRRRRRRR */
```

Comment:
Love the note left for the parts you are working with it makes it way easier later on or for the next developer to know what's what and where they are at much quicker.